### ISSUE TITLE
Incorrect behaviour on multi-value variables

### ISSUE THREAD

**Issue by rgualan** [grafana/grafana#25036]
**What happened**: 
When selecting query-based multi-value variables, the ids are shown instead of the labels.

![imagen](https://user-images.githubusercontent.com/22619017/82720876-39a23c00-9c7d-11ea-9440-df21b053ac8b.png)

**What you expected to happen**:
When selecting query-based multi-value variables, the labels should be shown (as in version 6).

![imagen](https://user-images.githubusercontent.com/22619017/82720923-a87f9500-9c7d-11ea-987a-af764e139a66.png)

**How to reproduce it (as minimally and precisely as possible)**:

- Create a query-based multi-value variable with a simple query as follows:
```select id as __value, name as __text from host```
- Test the combo

**Environment**:
- Grafana version: 7.0.0
- Data source type & version: Postgres v12
- OS Grafana is installed on: Ubuntu 18.04
- User OS & Browser: Ubuntu 18.04 & Firefox
- Grafana plugins: None
- Others: None

**Comment by baur** [comment]
Yes, I confirm a bug

**Comment by tskarhed** [comment]
What does the query response look like?

**Comment by rgualan** [comment]
The query response is as follows:

__value,__text
1,host-a
2,host-b

**Comment by baur** [comment]
![image](https://user-images.githubusercontent.com/2898361/85976761-0762c780-b9fd-11ea-914f-31927dbec96d.png)
Shows **__value** instead of **__text**

**Comment by baur** [comment]
steps to reproduce 

step1
create variable
```
select 1 as __value, 'text1' as __text
union all
select 2, 'text2'
union all
select 3, 'text3'
```
step2
Multi-value = true

step3
on dashboard select more one variable

![image](https://user-images.githubusercontent.com/2898361/85978142-e0f25b80-b9ff-11ea-869d-b28e8463d726.png)

**Comment by hugohaggmark** [comment]
@rgualan @baur I'm going to close this issue as we have fixed this in > 7.1.5, let me know if this is not the case so we can reopen this issue.

All the best
